opennms-base-assembly/src/main/filtered/bin/send-event.pl

```diff
@@ -163,11 +163,12 @@
 
 my ($sec,$min,$hour,$mday,$mon,$year,$wday,$yday,$isdst) = gmtime(time);
 $year += 1900;
-my $month = sprintf("%02d", $mon + 1);
-$mday     = sprintf("%02d", $mday);
-$min      = sprintf("%02d", $min);
-$sec      = sprintf("%02d", $sec);
-
+my $month = $mon;
+$hour = sprintf("%02d", $hour);
+$min = sprintf("%02d", $min);
+$sec = sprintf("%02d", $sec);
```

Comment:
This is the only change that is required for this.  line 164 gets gmtime so there should be no need to be concerned about timezones as internally, the script is working in GMT.